### PR TITLE
 🐛 make sure apb works with the latest version of the app

### PR DESCRIPTION
**Pull request by wei-lee**
To verify:

1. Build the apb and push to a registry
2. Make sure the ansible-serivice-broker can load apbs from the registry this APB is published to.
3. Make sure the service catalog is updated to load the latest apb
4. try provision and service and login. After login you should see a blank page (the service's docker image needs to be updated)
5. try deprovision and it should also work.

### CODE REVIEW

**Comment by wei-lee** [comment]
@secondsun mind taking a look?

File: roles/provision-mcs-server-apb/tasks/provision-service-server.yml [hunk 1]
```diff
@@ -34,12 +36,13 @@
...
-      - --openshift-service-account={{ proxy_serviceaccount_name }}
-      - --upstream=http://localhost:{{ mobile_client_service_port }}
-      - --openshift-sar={"namespace":"{{ namespace }}","resource":"deploymentconfigs","name":"mcs-server","verb":"update"}
+      - --openshift-service-account={{ mobile_client_service_sa_name }}
+      - --upstream=http://localhost:{{ mobile_client_service_port }}/
+      - --openshift-sar=[{"namespace":"{{ namespace }}","resource":"secrets","name":"mcs-server","verb":"list"}, {"namespace":"{{ namespace }}","resource":"buildconfig","name":"mcs-server","verb":"create"}]
       - --http-address=0.0.0.0:{{ mobile_client_service_proxy_port }}
       - --https-address=
       - --cookie-secret=SECRET
+      - --cookie-httponly=false
```
Comment (wei-lee) [inline roles/provision-mcs-server-apb/tasks/provision-service-server.yml:45]: this is required to make sure the cookies are accessible by the javascript

**Comment by secondsun** [comment]
Looking

**Comment by secondsun** [comment]
Everythign works as described.  Merging

**Pull request merged**